### DISCUSSION TITLE
V5.3 Bootstrap.min.css shows: "Invalid URL. The URL structure is /npm/package@version/file.js"

### DISCUSSION THREAD

**Discussion by majeam14965** [twbs/bootstrap#39788] [Q&A]
Hi,
After upgrading to V5.3,  Bootstrap.min.css shows: "Invalid URL. The URL structure is /npm/package@version/file.js" instead of the CSS file. 
See capture attached.

<img width="1110" alt="Captura de pantalla 2024-03-16 a las 9 29 55" src="https://github.com/twbs/bootstrap/assets/163635797/88d2d0b1-2516-4a1e-a8be-d9cd9a7af968">

**Comment by julien-deramond** [comment]
> After upgrading to V5.3,

From which version?

> Bootstrap.min.css shows: "Invalid URL. The URL structure is /npm/package@version/file.js" instead of the CSS file.

It hasn't changed on NPM side specifically for the v5.3, how do you install Bootstrap in your project?

**Reply by majeam14965** [reply]
I include it via CDN linking css and js

**Reply by majeam14965** [reply]
upgraded from 4.3.1

**Reply by majeam14965** [reply]
bootstrap.min.js shows the same message

**Reply by julien-deramond** [reply]
In 4.3.1 it was:

```html
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
<script src="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
```

In 5.3.3 it's:

```html
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.min.js" integrity="sha384-0pUGZvbkm6XF6gxjEnlmuGrJXVbNuzT9qBBavbLwCsOGabYfZo0T0to5eqruptLy" crossorigin="anonymous"></script>
```

The structure hasn't changed.
Maybe the integrity values are not the right ones on your side.
Or you can try with [alternative CDNs](https://getbootstrap.com/docs/5.3/getting-started/download/#alternative-cdns); in some countries, jsDelivr can be blocked.

**Reply by majeam14965** [reply]
The link and script you copied show the same message ( I copied them from the https://getbootstrap.com/docs/5.3/getting-started/introduction/#quick-start). 
In version 4.3.1 works fine with: 

```html
<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">

<script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.bundle.min.js" integrity="sha384-Piv4xVNRyMGpqkS2by6br4gNJ7DXjqk09RmUpJ8jgGtD7zP9yug3goQfGII0yAns" crossorigin="anonymous"></script>
```

**Reply by julien-deramond** [reply]
TBH IDK what's the problem on your side based on the information you gave, CDN version works as usual >> see https://codepen.io/julien-deramond/pen/vYMyXJK.

**Comment by majeam14965** [comment]
<img width="973" alt="Captura de pantalla 2024-03-16 a las 9 59 10" src="https://github.com/twbs/bootstrap/assets/163635797/c81cc097-f8b9-41a7-be76-f7fce93d29f7">
